src/object.js

```diff
@@ -5,7 +5,7 @@
 // Object borrows many conventions and utilities from Backbone.
 Marionette.Object = function(options) {
   this.options = _.extend({}, _.result(this, 'options'), options);
-
+  this.cid = _.uniqueId(this.cidPrefix || 'mno');
```

Comment:
`cidPrefix: 'mno'`

would work as well as the subclass will override that property